src/main/resources/entityResources/localization/eng/EntityMod-Card-Strings.json

```
@@ -96,6 +100,11 @@
     "NAME": "Parry",
     "DESCRIPTION": "Can only be played if every card in your hand is an Attack. Gain !B! Block."
   },
+  "entity:Ripple": {
+    "NAME": "Ripple",
+    "DESCRIPTION": "Apply 1 flux to all enemies.",
+    "UPGRADE_DESCRIPTION": "Apply 1 flux to all enemies. NL Draw a card."
```

change flux to entity:Flux on both DESCRIPTION, UPGRADE_DESCRIPTION.
```suggestion
    "UPGRADE_DESCRIPTION": "Apply 1 entity:Flux to all enemies. NL Draw a card."
```